Extension.php

```diff
@@ -15,6 +17,19 @@ public function register()
 
         $aliasLoader = AliasLoader::getInstance();
         $aliasLoader->alias('Location', Facades\Location::class);
+        
+        Relation::morphMap([
+            'reviews' => 'Igniter\Local\Models\Reviews_model',
+        ]);
+        
+        Orders_model::extend(function ($model) {
+            $model->relation['morphMany']['review'] = ['Igniter\Local\Models\Reviews_model'];
+        });
+        
+        Locations_model::extend(function ($model) {
+            $model->relation['hasMany']['reviews'] = ['Igniter\Local\Models\Reviews_model'];
+            $model->allowedSortingColumns = array_merge($model->allowedSortingColumns, ['reviews_count asc', 'reviews_count desc',]);
```

Comment:
Was wondering why you made the property public.. 😎 👍

Reply:
Yeah it wasn't enough - I've had to make some other changes which you may not like 😁 
Let me know if there is a better way.

Reply:
Yep, you can leave it as static and merge the array outside the extend closure/callback.